### PR TITLE
Create czsheets-Durham.topojson

### LINKED ISSUE #179: Some data from Southern US
## Adding Durham, NC to the map

Hi everyone! This issue is to track the addition of Durham, NC to the map, which can be found at:

 - Longitude: -78.898619
 - Latitude: 35.994033

### PR DESCRIPTION
**Pull request by czsheets**
Added my location to the map.

@githubteacher

closes #179

### CODE REVIEW

File: czsheets-Durham.topojson [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Add point for Durham, NC
```
Comment (SPCOxion) [inline czsheets-Durham.topojson]: This line is causing an error. Removing it should allow the file to work correctly

**Comment by vinay-x** [comment]
You should update the your branch as master branch has been changed.
And try again.

**Comment by hollenberry** [comment]
Hey @czsheets ! Friendly reminder ping that you have a line comment above! Once you follow the advice from @SPCOxion, I think that you should be good to 🚢 !

**Comment by czsheets** [comment]
Thanks!  I can see how to comment on my pull request, but how do I get back
to the original file to edit it?

**Comment by hollenberry** [comment]
@czsheets No problem! 😄 Attaching two screenshots below. Once you click on that pencil icon you should be able to make direct edits to your markdown file. 👍 

screen shot 2016-08-12 at 12 00 42 pm
screen shot 2016-08-12 at 12 00 45 pm

**Comment by hollenberry** [comment]
Hey @czsheets! Just a heads up, I'm planning on closing this out by the end of the week if it's still open, to keep the repo tidy. 

Feel free to reopen if you don't get a chance to merge this in time and want to return in the future.

**Pull request closed without merging**